web/src/layouts/bank/pages/dashboard/components/TransactionItem.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import { Avatar, AvatarFallback } from '@/components/ui/avatar';
```

Comment:
You're removing components that are in use? 🤨

Reply:
It's not used anymore on my version since we can't really rely on a user, i display the reason instead and no need of the avatar component

Reply:
It's used here: <https://github.com/overextended/ox_banking/pull/7/files#diff-a3504dea90e36b50bc1eb22195f37c4132a1ac55413ccf40f4e6ec6eb483de08R16-R18>

But yeah, instead of doing names we should do an icon depending if the transaction is inbound or outbound.

Reply:
idk what happen but this file is the old version, will update it directly

Reply:
yeah feel free to

Reply:
Done 
![image](https://github.com/overextended/ox_banking/assets/11323987/4bc2e772-5a4d-46c3-90d5-fd5370fe1cd1)